react/src/components/FolderInvitationResponseModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { useVFolderInvitations } from '../hooks/backendai';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import VFolderPermissionCell from './VFolderPermissionCell';
+import { FolderOutlined } from '@ant-design/icons';
+import { List, Button, Typography, theme, Descriptions } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface InvitationItem {
+  id: string;
+  vfolder_id: string;
+  vfolder_name: string;
+  invitee_user_email: string;
+  inviter_user_email: string;
+  mount_permission: string;
+  created_at: string;
+  modified_at: string | null;
+  status: string;
+  perm: string;
+}
+
+interface FolderInvitationResponseModalProps extends BAIModalProps {
+  onRequestClose?: () => void;
+}
+
+const FolderInvitationResponseModal: React.FC<
+  FolderInvitationResponseModalProps
+> = ({ onRequestClose, ...baiModalProps }) => {
+  const { token } = theme.useToken();
+  const { t } = useTranslation();
+  const [
+    { invitations },
+    { acceptInvitation, rejectInvitation: declineInvitation },
+  ] = useVFolderInvitations();
+
+  return (
+    <BAIModal
+      onCancel={onRequestClose}
+      title={t('data.InvitedFolders')}
+      footer={null}
+      {...baiModalProps}
+    >
+      <List
+        dataSource={invitations}
+        renderItem={(item: InvitationItem) => (
```

Comment:
Use useCallback for re-rendering ListItem in case of performance. See https://github.com/lablup/backend.ai-webui/blob/main/react/src/components/Chat/VirtualChatMessageList.tsx#L67